src/stats.js

```diff
@@ -27,50 +27,12 @@ const self = (module.exports = {
         })
     );
 
-    const result = statsResult.reduce((a, b) => {
-      return {
-        readyForReview: {
-          fromPrCreation: averageTimeDiff(
-            a.readyForReview.fromPrCreation,
-            b.readyForReview.fromPrCreation
-          ),
-          fromPreviousStep: averageTimeDiff(
-            a.readyForReview.fromPreviousStep,
-            b.readyForReview.fromPreviousStep
-          ),
-        },
-        firstReview: {
-          fromPrCreation: averageTimeDiff(
-            a.firstReview.fromPrCreation,
-            b.firstReview.fromPrCreation
-          ),
-          fromPreviousStep: averageTimeDiff(
-            a.firstReview.fromPreviousStep,
-            b.firstReview.fromPreviousStep
-          ),
-        },
-        firstApprovedReview: {
-          fromPrCreation: averageTimeDiff(
-            a.firstApprovedReview.fromPrCreation,
-            b.firstApprovedReview.fromPrCreation
-          ),
-          fromPreviousStep: averageTimeDiff(
-            a.firstApprovedReview.fromPreviousStep,
-            b.firstApprovedReview.fromPreviousStep
-          ),
-        },
-        merged: {
-          fromPrCreation: averageTimeDiff(
-            a.merged.fromPrCreation,
-            b.merged.fromPrCreation
-          ),
-          fromPreviousStep: averageTimeDiff(
-            a.merged.fromPreviousStep,
-            b.merged.fromPreviousStep
-          ),
-        },
-      };
-    });
+    const result = {}
+
+    ["readyForReview", "firstReview", "firstApprovedReview", "merged"].forEach(column => result[column] = {
```

Comment:
Nice clean up 🙏🏻

Reply:
Sure 🙂 For some reason Prettier wasn't working on my end so we might clean it up even more

Reply:
I put the same prettier and eslint as main wooclap, let's merge and I'll iterate on some other stuff